tests/cssParser.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
```

Comment:
What's the reason you're not using `jest` here? Seems cleaner than these manual asserts?

Reply:
Because I didnt even include a build chain for this project. Decided to keep it really simple and barebone in order to to setup my Babel configs till 2021

Reply:
I mean you did hook it into travis with a test script from what I can tell? :wink: